### PR TITLE
hotfix of readme

### PR DESCRIPTION
**Pull request by kretatusha**
refactor readme

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 1.0.1
```
Comment (SharUpOff) [inline CHANGELOG.md:3]: I think there's no CHANGELOG and no release of a new version required for any README changes

Comment (kretatusha) [reply CHANGELOG.md:3]: I don't know about changelog, but a new release would refactor an information on pypi.org (https://pypi.org/project/http-toolkit/)

Comment (SharUpOff) [reply CHANGELOG.md:3]: Oh, you're right.

**Review by SharUpOff — APPROVED** [review]
Have no more questions :)

**Pull request merged**